acs-integration-tests/src/test/java/com/ge/predix/cloud/foundry/application/ApplicationHelper.java

```diff
@@ -0,0 +1,353 @@
+
+        private static final String ACS_SERVICE_ID = "predix-acs";
+
+        private static final Map<String, String> COMMON_ENVIRONMENT_VARIABLES = new HashMap<String, String>() {{
+            put("ACS_BASE_DOMAIN",
+                    String.format("%s-integration.%s", ACS_SERVICE_ID, System.getenv("CF_BASE_DOMAIN")));
+            put("ACS_DB", CloudFoundryUtilities.Acs.ACS_DB_SERVICE_INSTANCE_NAME);
+            put("ACS_DEFAULT_ISSUER_ID", System.getenv("ACS_DEFAULT_ISSUER_ID"));
+            put("ACS_REDIS", CloudFoundryUtilities.Acs.ACS_REDIS_SERVICE_INSTANCE_NAME);
+            put("ACS_SERVICE_ID", ACS_SERVICE_ID);
+            put("NUREGO_API_KEY", System.getenv("NUREGO_API_KEY"));
+            put("NUREGO_API_URL", System.getenv("NUREGO_API_URL"));
+            put("NUREGO_BATCH_MAX_MAP_SIZE", "1");
+            put("UAA_CHECK_HEALTH_URL", System.getenv("UAA_CHECK_HEALTH_URL"));
+            put("ZAC_CLIENT_ID", System.getenv("ZAC_CLIENT_ID"));
+            put("ZAC_CLIENT_SECRET", System.getenv("ZAC_CLIENT_SECRET"));
+            put("ZAC_UAA_URL", System.getenv("ZAC_UAA_URL"));
+            put("ACS_UAA_URL", System.getenv("ACS_UAA_URL"));
+            put("ZAC_URL", System.getenv("ZAC_URL"));
+            put("ENABLED_REDIS_HEALTH_CHECK", "true");
+            put("DEPLOYMENT_TYPE", System.getenv("DEPLOYMENT_TYPE"));
+            put("BASE_APP_NAME", System.getenv("BASE_APP_NAME"));
+            put("CONFIG_CLIENT_ID", System.getenv("CONFIG_CLIENT_ID"));
+            put("CONFIG_CLIENT_SECRET", System.getenv("CONFIG_CLIENT_SECRET"));
+            put("CONFIG_SERVER_URI", System.getenv("CONFIG_SERVER_URI"));
+            put("CONFIG_OAUTH_TOKEN_URI", System.getenv("CONFIG_OAUTH_TOKEN_URI"));
+            put("ENCRYPTION_KEY", System.getenv("ENCRYPTION_KEY"));
+            put("AUDIT_SERVICE_NAME", System.getenv("AUDIT_SERVICE_NAME"));
+            put("AUDIT_UAA_URL", System.getenv("AUDIT_UAA_URL"));
+            put("AUDIT_UAA_CLIENT_ID", System.getenv("AUDIT_UAA_CLIENT_ID"));
+            put("AUDIT_UAA_CLIENT_SECRET", System.getenv("AUDIT_UAA_CLIENT_SECRET"));
+            put("SPRING_PROFILES_ACTIVE", System.getenv("SPRING_PROFILES_ACTIVE"));
+            put("JAVA_OPTS", System.getenv("JAVA_OPTS"));
+        }};
+
+        void pushApplication(final ApplicationHelper applicationHelper,
+                final Map<String, String> additionalEnvironmentVariables) throws Exception {
+
+            Map<String, String> environmentVariables = new HashMap<>(COMMON_ENVIRONMENT_VARIABLES);
+            environmentVariables.putAll(additionalEnvironmentVariables);
+            environmentVariables.values().removeIf(Objects::isNull);
+
+            CloudFoundryService postgresService = new CloudFoundryService("shared-nr",
+                    CloudFoundryUtilities.Acs.ACS_DB_SERVICE_INSTANCE_NAME, "postgres", Collections.emptyMap());
+            CloudFoundryService redisService = new CloudFoundryService("shared-vm",
```

Comment:
We could but I don't think we're changing these plans anytime soon? Once we do we can pull it out then?